package.json

```diff
@@ -7,7 +7,9 @@
     "dev": "node server.js",
     "build": "next build",
     "start": "NODE_ENV=production node server.js",
-    "deploy": "now --public"
+    "deploy": "now --public",
+    "prettier": "prettier --config .prettierrc --write 'components/**/*.js' 'handlers/**/*.js' 'lib/**/*.js' 'pages/**/*.js' 'server.js'",
```

Comment:
Any reason we can't do `*.js {components,handlers,lib,pages}/*.js` here?

Reply:
That'd be smart! Good idea!